### DISCUSSION TITLE
Where to find version number (docker)?

### DISCUSSION THREAD

**Discussion by ltguillaume** [HemmeligOrg/Hemmelig.app#252] [Q&A] [answered]
(When running a docker image of Hemmelig,) is there a place where I can find the current version number, preferably just in the web interface?

**Comment by bjarneo** [comment, marked as answer]
@ltguillaume if you look at the release tab, it is 1:1 mapped with the docker version. https://github.com/HemmeligOrg/Hemmelig.app/tags

i.e. hemmeligapp/hemmelig:v5.16.5

Of course, having a script updating the readme would be the best approach.

**Reply by ltguillaume** [reply]
Yeah I just found myself wanting to know the current version I was running in order to check out the commits that came after it.

**Comment by bjarneo** [comment]
The version is now applied to the instance (footer) and nav menu (account page and what not) for the docker images.
